oauth2_http/javatests/com/google/auth/TestUtils.java

```diff
@@ -9,9 +9,12 @@
 import com.google.common.base.Splitter;
 import com.google.common.collect.Lists;
 
+import java.lang.RuntimeException;
```

Comment:
Thanks for catching this error. It was left over from a previous helper. I'll remove.